precompiles/staking/StakingI.sol

```diff
@@ -139,7 +139,7 @@ interface StakingI is authorization.AuthorizationI {
         CommissionRates calldata commissionRates,
         uint256 minSelfDelegation,
         address delegatorAddress,
-        string memory validatorAddress,
+        address validatorAddress,
```

Comment:
I agree it would make sense to remove the `delegaterAddress` since it's going to be the same address 👍